websites/F/Facepunch Wiki/dist/metadata.json

```diff
@@ -0,0 +1,41 @@
+{
+  "$schema": "https://schemas.premid.app/metadata/1.0",
+  "author": {
+    "name": "MatStef132",
+    "id": "269787765120565248"
+  },
+  "url": "wiki.facepunch.com",
+  "description": {
+    "en": "A wiki for facepunch games, which includes s&box, Garry's Mod and Rust"
+  },
+  "service": "Facepunch Wiki",
+  "version": "1.0.0",
+  "logo": "https://imgur.com/7J6xKBQ.png",
+  "thumbnail": "https://imgur.com/dkTbScQ.png",
```

Comment:
upload a better logo
and thumbnail should be a promotional banner or a screenshot of the website homepage